lib/icmp/icmpv4_packet.mli

```diff
@@ -25,14 +24,11 @@ end
 module Marshal : sig
   type error = string
 
-  val echo_request : buf:Cstruct.t -> payload:Cstruct.t ->
-    id:Cstruct.uint8 -> seq:Cstruct.uint8 -> (unit, error) Result.result
-
-  val echo_reply : buf:Cstruct.t -> payload:Cstruct.t ->
-    id:Cstruct.uint8 -> seq:Cstruct.uint8 -> (unit, error) Result.result
-
-  val would_fragment : buf:Cstruct.t -> ip_header:Cstruct.t ->
-    ip_payload:Cstruct.t -> next_hop_mtu:Cstruct.uint16 -> (unit, error) Result.result
```

Comment:
why did those functions disappear from the mli? if they are not needed, maybe also remove them from the ml?